css/style.css

```diff
@@ -0,0 +1,34 @@
+body{
+  /*text-align: center;*/
+  color:white;
+      background-image: linear-gradient(
+      115deg,
+      rgba(58, 58, 158, 0.8),
+      rgba(136, 136, 206, 0.7)
+    ),
+    url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+}
+form{
+  width: 640px;
+  background-color: rgb(39, 38, 69);
+  color:white;
+  margin: 0 auto;
+  padding: 30px;
+}
+#name, #email, #number, select{
```

Comment:
Nice job with your styling! Please remember that if more than 1 ids need the same styling, you can always create a class for all of the elements to only create one styling rule. Please make the changes accordingly.